manifests/rabbitmq/monitoring.pp

```diff
@@ -0,0 +1,32 @@
+class profiles::rabbitmq::monitoring (
+  String  $hostname          = 'orange-gnu.rmq.cloudamqp.com',
+  Integer $port              = 443,
+  Boolean $use_ssl           = true,
+  String  $check_interval    = '15s',
+  String  $exchanges_regexes = '[".*"]',
+  String  $queues_regexes    = '[".*"]',
+  String  $vhosts,           # example value: '["udb3","uitid-prod","mailjet-prod","mpm-prod","projectaanvraag-prod"]'
```

Comment:
Ik zou deze misschien laten defaulten naar een lege array. Anders is het een verplichte parameter en moet hij eigenlijk vooraan in de parameterlijst staan